sdk/communication/azure-communication-administration/tests/test_phone_number_administration_client.py

```diff
@@ -125,6 +127,15 @@ def setUp(self):
             self.capabilities_id = "capabilities_id"
             self.release_id = "release_id"
 
+    @pytest.mark.live_test_only
+    @pytest.mark.skipif(SKIP_PHONE_NUMBER_TESTS, reason=PHONE_NUMBER_TEST_SKIP_REASON)
+    def test_list_all_phone_numbers_from_managed_identity(self):
+        endpoint, access_key = parse_connection_str(self.connection_str)
+        credential = create_token_credential()
+        phone_number_client = PhoneNumberAdministrationClient(endpoint, credential)
+        pages = phone_number_client.list_all_phone_numbers()
+        assert pages.next()
```

Comment:
should we check if the # of phone numbers we get match with what we expected?

Reply:
We don't know which (or how many) phone numbers we're going to receive exactly, that's why this test only checks that we indeed returned something. This test is also copied from the current list_all_phone_numbers test from this same file, just with the added managed identity authentication.